ci/test/06_script_b.sh

```diff
@@ -8,6 +8,9 @@ export LC_ALL=C.UTF-8
 
 cd "build/bitcoin-$HOST" || (echo "could not enter distdir build/bitcoin-$HOST"; exit 1)
 
+mkdir -p contrib
+cp -R ../../contrib/ contrib/
```

Comment:
Also, if those files are missing because they are not in the distdir, I'd prefer to add them instead of copying them manually in the ci

Reply:
Yes, that seems cleaner, doing that instead now